src/inference/Inference.jl

```diff
@@ -250,6 +282,9 @@ end
 # VarInfo, combined with spl.info, to Sample
 function Sample(vi::AbstractVarInfo, spl::Sampler)
     s = Sample(vi)
+    if haskey(spl.info, :adaptor)
```

Comment:
Is it necessary to make this? It seems it would be better if we don't know about AHMC at this point?

Reply:
This appears to be some leftover code, I'll remove this shortly.